gef-ui/server/controllers/facilities/index.js

```diff
@@ -54,6 +54,7 @@ const createFacility = async (req, res) => {
       });
 
       return res.render('partials/facilities.njk', {
+        facilityType: facilityTypeString,
```

Comment:
Can the sending variable be called `facilityType` to avoid introduction of `facilityTypeString` variable?

Reply:
it wouldn't work as only contingent comes through as facilityType.  line 45 then checks if defined or else sets it to cash.  Then it's converted to lowercase where the variable facilityTypeString is set

Reply:
@IgnatG @ttbarnes are you fine with the changes I have made here?